Resources/Prototypes/_DV/Entities/Objects/Tools/jaws_of_life.yml

```diff
@@ -0,0 +1,56 @@
+- type: entity
+  abstract: true
+  parent: BaseItem
+  id: BaseJawsOfLife
+  components:
+  - type: Tag
+    tags:
+    - JawsOfLife
+  - type: Sprite
+    sprite: Objects/Tools/jaws_of_life.rsi
+    state: jaws_pry
+  - type: Item
+    size: Normal
+  - type: Clothing
+    sprite: Objects/Tools/jaws_of_life.rsi
+    quickEquip: false
+    slots:
+    - Belt
+  - type: ToolTileCompatible
+  - type: Tool
+    qualities:
+    - Prying
+    useSound: /Audio/Items/jaws_pry.ogg
+  - type: Prying
+    speedModifier: 1.5
+    pryPowered: true
+    useSound: /Audio/Items/jaws_pry.ogg
+  - type: MeleeWeapon
+    wideAnimationRotation: 90
+    damage:
+      types:
+        Blunt: 10
+    soundHit:
+      collection: MetalThud
+
+- type: entity
+  name: emergency jaws of life
+  parent: [BaseJawsOfLife, PowerCellSlotMediumItem]
+  id: EmergencyJawsOfLife
+  description: A set of Jaws of Life. EMTs may use them to pry open powered airlocks in an emergency.
+  components:
+  - type: Sprite
+    sprite: _DV/Objects/Tools/emergency_jaws_of_life.rsi
+    state: jaws_pry
+  - type: Clothing
+    sprite: _DV/Objects/Tools/emergency_jaws_of_life.rsi
+    quickEquip: false
+    slots:
+    - Belt
+  - type: ToolTileCompatible
+  - type: Prying
+    speedModifier: .75 # High torque low speed, Half speed of proper JoL
```

Comment:
thus is insanely slow for powered doors, any half decent ai would be with you much faster